content/sql/concepts/aggregate-functions/terms/count/count.md

```diff
@@ -25,7 +25,7 @@ FROM table_name;
 
 `COUNT()` is a function that takes the name of a column as an argument and counts the number of rows where the column is not `NULL`.
 
-**Note:** A column name of the table can also be used instead of `*`. Unlike `COUNT(*)`, this variation `COUNT(column_name)` will not count `NULL` values in that column.
+**Note:** There are different variations to count records.  `COUNT(*)` counts number of rows including `NULL` values. `COUNT(1)` counts non `NULL` values. `COUNT(*)` and `COUNT(1)` have the same results because `COUNT(1)` replaces all records with the value `1` and counts the number of value `1`. The `COUNT(column_name)` counts non `NULL` values in that column.
```

Comment:
What does COUNT(1) replace with the value 1 (Please include this)?

Can we also refer to the variations as what's passed in (like * or 1 rather than the entire COUNT() func) and reword accordingly.

Let's also take this out of a note and place the points in bullet format (line 26 can be part of these bullet points).

Reply:
@yangc95 
Thank you for your feedback.  

I'm not sure if I understand your question: 
> What does COUNT(1) replace with the value 1 (Please include this)
Could you please clarify this?

I reordered examples and updated with bullet format.

Reply:
Yeah! Does it replace null values with 1?

Reply:
@yangc95 
Yes!  Thank you for your help, I see what you mean.
It replaces all values including null with 1, so it counts all records including nulls.
I'll clarify that in the document.  Thanks!

Reply:
@yangc95 
I just modified the document and pushed the sql-count-branch.  Could you please review it when you have a chance? Thanks!